VATSpy.dat

```diff
@@ -17675,7 +17675,6 @@ ZYQQ|Qiqihar|47.38306|123.91639||ZYSH|0
 ZYTL|Dalian|38.965556|121.538333||ZYSH|0
 ZYTX|Shenyang|41.638333|123.485||ZYSH|0
 ZYYJ|Yanji|42.8828|129.451258||ZYSH|0
-

```

Comment:
typo or intentional?

Reply:
Not sure what happened I think it was a typo and will be updated @NelisV